apps/generator/test/test-templates/react-template/template/test-file.md.js

```diff
@@ -1,11 +1,25 @@
 import { File, Text } from '@asyncapi/generator-react-sdk';
+import PropTypes from 'prop-types';
 
-export default function({ asyncapi, params }) {
+const TestFile = ({ asyncapi, params }) => {
   return (
     <File name="test-file.md">
       <Text>This is a markdown file for my application.</Text>
       <Text>App name is: **{ asyncapi.info().title() }**</Text>
       <Text>Version {params.version} running on {params.mode} mode </Text>
     </File>
   );
-}
\ No newline at end of file
+}
```

Comment:
How's about using `function` syntax as it was used before in place of arrow functions since this will maintain the code consistency of the codebase.

Reply:
Implemented the feature using the 'function' keyword